include/libkernelflinger/log.h

```diff
@@ -38,9 +38,13 @@
 #include <vars.h>
 
 EFI_STATUS log_flush_to_var(BOOLEAN nonvol);
+EFI_FILE *efi_log_file;
```

Comment:
This should be a `static` variable inside the `log.c` module.  And the variable name should then be simplified to `file`.

Reply:
Already modified